### PR TITLE
Calculations and run plan for OAV loop centring

### LINKED ISSUE #421: Fix  bugs from oav centring
When testing oav centring we encountered bugs:
* The oav centring would appear to be working, then the pin would suddenly veer off way too far (+y_pixels, or +x in the beamline coordinate system).

Testing criteria:
* Ensure these features work on next  test.

### PR DESCRIPTION
**Pull request by evalott100**
Fixes #421  #420 

### To test:
1. Run against s03 and the branch of issue 26 in tickit_devices
2. Check output is acceptable.

### CODE REVIEW

File: src/artemis/devices/unit_tests/test_oav_centring.py [hunk 1]
```diff
@@ -1,32 +1,335 @@
...
+        mock_parameters.micronsPerYPixel,
+    )
+    expected_values = expected_values * 1e-3
+    expected_values[0] /= mock_parameters.micronsPerXPixel
+    expected_values[1] /= mock_parameters.micronsPerYPixel
+    expected_values[2] /= mock_parameters.micronsPerYPixel
+    expected_values = np.around(expected_values, decimals=5)
+
+    assert np.array_equal(np.around(results, decimals=5), expected_values)
+"""
```
Comment (DominicOram) [inline src/artemis/devices/unit_tests/test_oav_centring.py:335]: Should: Can you either reinstate or remove this?

**Review by DominicOram — CHANGES_REQUESTED** [review]
Short review at a glance: This has changes that are unrelated and were done as hotfixes on the beamline, can you clean the branch up to be just the changes relevant to the issue?  I think this should clean up most of the merge conflicts but if it doesn't can you clean up those too? Additionally, given we have a tickit device for this can you add a system test for it?

File: src/artemis/devices/unit_tests/test_oav_centring.py (hunk 1 above)
Comment (evalott100) [reply src/artemis/devices/unit_tests/test_oav_centring.py:338]: I'll start writing writing the system tests now. Should be a seperate branch.

**Comment by evalott100** [comment]
> Short review at a glance: This has changes that are unrelated and were done as hotfixes on the beamline, can you clean the branch up to be just the changes relevant to the issue? I think this should clean up most of the merge conflicts but if it doesn't can you clean up those too? Additionally, given we have a tickit device for this can you add a system test for it?

* I'll do system tests next (on another branch)
* So I'm a little confused, the initial pull issue I was tackling was 333_ run_calculations on edge waveforms. After merging that into the v0.9.10. branch for testing, I had fixes - which I applied to this new branch. Should I merge branch 421 into 333 first, then merge 333 into main? We can't merge fixes to calculations to main if those changes don't already exist on main.

**Comment by evalott100** [comment]
#456 This pull request contains a branch which doesn't have other code.

**Pull request closed without merging**